# advertises finding relatives to aruco markers
bool[] findings